src/input/vita.c

```diff
@@ -434,19 +436,23 @@ static inline void vitainput_process(void) {
       front_state = ON_SCREEN_SWIPE;
       break;
     case ON_SCREEN_SWIPE:
-      if (touch.finger > 0) {
-        switch (touch.finger) {
-          case 1:
-            move_mouse(swipe, touch);
-            break;
-          case 2:
-            move_wheel(swipe, touch);
-            break;
-        }
-        memcpy(&swipe, &touch, sizeof(swipe));
-      } else {
-        front_state = NO_TOUCH_ACTION;
+      if (drag_active != new_drag_status) {
+        mouse_click(1, new_drag_status);
+        drag_active = new_drag_status;
+      }
+      switch (touch.finger) {
+        case 1:
+        case 3:
+          move_mouse(swipe, touch);
+          break;
```

Comment:
imo, i used wrong approach for mouse actions.
could you make the new config for the changing finger - mouse mapping?

I also planed implement touch screen action, but it requires additional work
for example I want
1. 1 finger can choose mouse move or touch screen action
2. other fingers can choose move cursor, wheel
3. should make the specialized action to print override button; we cannot use corner button in the touchscreen mode
    (for example swipe 4~5 finger or tap 4~5 finger)

so option could be
```
use touchscreen mode  < false >
finger
1 swipe      < move cursor >
  tap        < click left >
  double tap < none >
2 swipe      < move wheel >
  tap        < click right >
  double tap < none >
3 swipe      < toggle pannel >
  tap        < none >
  double tap < none >
4 swipe      < none >
  tap        < none >
  double tap < none >
5 swipe      < none >
  tap        < none >
  double tap < none >
```

Reply:
I actually have a [WIP touchscreen](https://github.com/MakiseKurisu/vita-moonlight/tree/TouchpadDraft) branch right now. I haven't messed with GUI yet, but both touchscreen and trackpad mode are coexisting, and user can switch between those 2 modes with a 6-finger tap. This way the existing code can be reused.
I also fixed some other mistakes regarding touchpad coordinate calculation there so you can take a look.

Reply:
Let me know if you still want me to create that config menu.